app/src/main/java/io/gnosis/safe/ui/StartActivity.kt

```diff
@@ -252,8 +235,14 @@ class StartActivity : BaseActivity(), SafeOverviewNavigationHandler, AppStateLis
         with(binding) {
             toolbarShadow.visible(false)
             chainRibbon.visible(true)
-            //TODO: get chain name and ribbon color from safe data
-            chainRibbon.text = BuildConfig.BLOCKCHAIN_NAME
+            safe.chain?.let {
+                chainRibbon.text = it.name
+                chainRibbon.setTextColor(Color.parseColor(it.textColor))
+                chainRibbon.setBackgroundColor(Color.parseColor(it.backgroundColor))
+            } ?: run {
+                chainRibbon.text = ""
+                chainRibbon.setBackgroundColor(getColorCompat(R.color.primary))
```

Comment:
So primary (green) is the default (and no chain name) if we don't have any chain info?

Reply:
At least we will have default values for mainnet that will be added in the next pr. Edge case when we don't have chain info for a safe should not be possible